src/System.IO.Packaging/tests/Tests.cs

```diff
@@ -41,6 +41,88 @@ public static FileInfo GetGuidNameForNewFile(string test, string extension)
         }
 
         [Fact]
+        public void T201_FileFormatException()
+        {
+            var e = new FileFormatException();
+            Assert.NotNull(e);
```

Comment:
This Assert isn't testing anything, other than the runtime correctly allocating an object for new.  If the goal here is to validate something about FileFormatException, we should be asserting things about the instance, such as making sure its properties map to what we expect, e.g.
```C#
Assert.Null(e.SourceUri);
```